submissions/vakulinina/memory-pair-game/main.js

```diff
@@ -0,0 +1,98 @@
+const cardBoard = document.querySelector('.card-board');
+const newGameButton = document.querySelector('.new-game-btn');
+const winPopup = document.querySelector('.win-popup');
+const cardIds = [
+  'paw-1',
+  'paw-2',
+  'paw-3',
+  'paw-4',
+  'paw-5',
+  'paw-6',
+  'paw-7',
+  'paw-8'
+];
+
+let firstFlip;
+let secondFlip;
+let pairsOpen = 0;
+
+const cards = cardIds.flatMap(id => [createCard(id), createCard(id)]);
+
+function createCard(id) {
+  let newCard = document.createElement('li');
+  newCard.classList.add('card');
+  newCard.id = id;
+  newCard.innerHTML = `
+    <div class="card-front">
+      <img src="img/icon-cat.svg" alt="">
+    </div>
+    <div class="card-back">
+      <img src="img/${id}.png" alt="">
```

Comment:
try to store full src in your obj/array (what if one image must be in .jpg format and another in .png ?)